TheFourFifths/consus
# Clock system tff 176

### Summary

The clock module that can be mocked out for time based tests and also will trigger the notification emails.

### Checklist


### How to Review

1. Ensure tests work
2. Ensure clock init code makes sense. Part if commented out as it waits to integrate with the email system.

### Links

- [TFF-176](https://msoese.atlassian.net/browse/TFF-176)
